### PR TITLE
Docstring

### PR DESCRIPTION
**Pull request by Cybsloth**
Add additional linters for docstrings: `darglint`, `flake8-docstring`.

### CODE REVIEW

File: lightautoml/dataset/roles.py [hunk 1]
```diff
@@ -130,7 +131,16 @@ def from_string(name: str, **kwargs: Any) -> "ColumnRole":
 
 
 class NumericRole(ColumnRole):
-    """Numeric role."""
+    """Numeric role.
```
Comment (dev-rinchin) [inline lightautoml/dataset/roles.py:134]: Use the text from the removed docstring "Create numeric role with specific numeric dtype."

File: lightautoml/pipelines/features/text_pipeline.py [hunk 2]
```diff
@@ -34,19 +34,11 @@
...
 
 class NLPDataFeatures:
-    """
-    Class contains basic features transformations for text data.
-    """
+    """Class contains basic features transformations for text data."""
 
     _lang = {"en", "ru", "multi"}
 
     def __init__(self, **kwargs: Any):
```
Comment (dev-rinchin) [inline lightautoml/pipelines/features/text_pipeline.py:41]: write about "default parameters for nlp pipeline constructor."

File: lightautoml/pipelines/features/text_pipeline.py [hunk 3]
```diff
@@ -34,19 +34,11 @@
 
 
 class NLPDataFeatures:
-    """
-    Class contains basic features transformations for text data.
-    """
+    """Class contains basic features transformations for text data."""
```
Comment (dev-rinchin) [inline lightautoml/pipelines/features/text_pipeline.py:37]: write about "default parameters for nlp pipeline constructor."

File: lightautoml/text/sentence_pooling.py [hunk 4]
```diff
@@ -1,5 +1,5 @@
 """Different Pooling strategies for sequence data."""
-
+# flake8: noqa
```
Comment (dev-rinchin) [inline lightautoml/text/sentence_pooling.py:2]: set exact rule to ignore

File: lightautoml/pipelines/selection/base.py [hunk 5]
```diff
@@ -212,11 +211,14 @@ def select(self, dataset: LAMLDataset) -> LAMLDataset:
 
     def map_raw_feature_importances(self, raw_importances: Series):
         """Calculate input feature importances.
+
         Calculated as sum of importances on different levels of pipeline.
 
         Args:
             raw_importances: Importances of output features.
 
+        # noqa: DAR201
```
Comment (dev-rinchin) [inline lightautoml/pipelines/selection/base.py:220]: Does it work?

File: lightautoml/automl/presets/utils.py [hunk 6]
```diff
@@ -1,3 +1,5 @@
+# flake8: noqa
```
Comment (dev-rinchin) [inline lightautoml/automl/presets/utils.py:1]: set the exact rule to ignore

File: lightautoml/addons/interpretation/l2x_model.py [hunk 7]
```diff
@@ -1,3 +1,4 @@
+# flake8: noqa
```
Comment (dev-rinchin) [inline lightautoml/addons/interpretation/l2x_model.py:1]: set the exact rule to ignore

File: lightautoml/addons/uplift/metrics.py [hunk 8]
```diff
@@ -182,15 +190,22 @@ def calculate_uplift_auc(
...
+        ) = calculate_min_max_uplift_auc(y_true, treatment, mode)
 
         uplift_auc = (uplift_auc - min_auc) / (max_auc - min_auc)
 
     return uplift_auc
 
 
-def calculate_min_max_uplift_auc(y_true: np.ndarray, treatment: np.ndarray, mode: str = "adj_qini"):
-    """Calculate AUC uplift curve for `base` and `perfect` models
+def calculate_min_max_uplift_auc(
```
[the remark this answers is not in the document]
Comment (Cybsloth) [reply lightautoml/addons/uplift/metrics.py:203]: Set arguments in one line.

File: lightautoml/addons/interpretation/l2x_model.py (hunk 7 above)
Comment (Cybsloth) [reply lightautoml/addons/interpretation/l2x_model.py]: Specified error codes in the configuration file.

File: lightautoml/pipelines/selection/base.py (hunk 5 above)
Comment (Cybsloth) [reply lightautoml/pipelines/selection/base.py:220]: Yes, so all errors in this line are disabled, but the error code is specified for users

File: lightautoml/text/sentence_pooling.py (hunk 4 above)
Comment (Cybsloth) [reply lightautoml/text/sentence_pooling.py]: Specified error codes in the configuration file.

File: lightautoml/automl/blend.py [hunk 9]
```diff
@@ -101,7 +101,7 @@ def _predict(self, predictions: Sequence[LAMLDataset]) -> LAMLDataset:
         Args:
             predictions: Sequence of predictions from pruned datasets.
 
-        Returns:
+        Returns:  # noqaL DAR202
```
[the remark this answers is not in the document]
Comment (Cybsloth) [reply lightautoml/automl/blend.py:104]: Fix.

File: lightautoml/addons/interpretation/l2x_model.py (hunk 7 above)
Comment (dev-rinchin) [reply lightautoml/addons/interpretation/l2x_model.py]: Please check "flake8: noqa" in all files. I think, I haven't highlighted all of them.

Comment (Cybsloth) [reply lightautoml/addons/interpretation/l2x_model.py]: fix

File: lightautoml/pipelines/features/text_pipeline.py (hunk 3 above)
Comment (Cybsloth) [reply lightautoml/pipelines/features/text_pipeline.py:37]: Create task on the kanban board.

**Pull request merged**